mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -1,12 +1,19 @@
-/**
- * MongoDB client binding for YCSB.
+/*
+ * Copyright (c) 2014, Yahoo!, Inc. All rights reserved.
```

Comment:
We don't have any kind of CLA that assigns copyrights to Yahoo!, so let's leave out adding these for now. I've been trying to think of the best way to handle #313 and all I've got so far is that adding Yahoo copyright statements isn't the way to go. :)

If they're already present, please leave them. That's a whole different mess.